service/dap/utils.go

```diff
@@ -0,0 +1,12 @@
+package dap
+
+type tuple struct {
+	a, b interface{}
+}
+
+var min = func(i, j int) int {
```

Comment:
Why not `func min(...)` ?

Reply:
I am not sure what you mean.

Reply:
It's not clear to me why this is:

```
var min = func(i, j int) int {
```

Instead of the more common (and prevailing throughout the rest of delve and delve/service/dap):

```
func min(i, j int) int {
```

Reply:
Also please add a function comment

Reply:
Added comment and changed signature format. The other one was there because this used to be a nested function in another function.